src/scss/components/ec-btn/ec-btn.story.js

```diff
@@ -4,7 +4,7 @@ import StoryRouter from 'storybook-vue-router';
 import EcIcon from '@/components/ec-icon';
 import { LIGHT_THEME, DARK_THEME } from '../../../../.storybook/backgrounds';
 
-const stories = storiesOf('Button', module);
+const stories = storiesOf('Button Styles', module);
```

Comment:
@mcibique @matallo @jaimeperezpedra As we have 3 different button stories I renamed the old one "Button styles" since is not a component, and the new one I named it "button component" so be a clear distinction of what is what. Another approach can be to completely delete the other story. Let me know your thoughts.

Reply:
LGTM, in the future we'll have Chameleon styles, and chameleon components which will be grouped separately in Storybook